Personal_Portofolio.html

```diff
@@ -22,20 +22,17 @@
 			
 		</nav>
 	</div>
```

Comment:
[OPTIONAL]
🔳 Please consider using the appropriate tags for each element, and stick the HTML5 tags (e.g., header, nav) over divs.